x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -157,7 +157,7 @@ export const useEndpointActionItems = (
                 children: (
                   <FormattedMessage
                     id="xpack.securitySolution.endpoint.actions.actionsLog"
```

Comment:
should probably update the translation keys

Reply:
updated